From d6fbd165ca96c4a797eb75efe2fab5162f0b7e8b Mon Sep 17 00:00:00 2001
From: clarencelol <clarencekuiek@proton.me>
Date: Wed, 11 Dec 2024 15:18:34 +0800
Subject: [PATCH 22/28] SystemUI: Adapt Smartspace A15-QPR1

* Building error text
error: BcSmartspaceView is not abstract and does not override abstract method setBgHandler(Handler) in SmartspaceView
public class BcSmartspaceView extends FrameLayout implements BcSmartspaceDataPlugin.SmartspaceTargetListener, BcSmartspaceDataPlugin.SmartspaceView {

Signed-off-by: clarencelol <clarencekuiek@proton.me>
---
 .../google/android/systemui/smartspace/BcSmartspaceView.java | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/packages/SystemUI/bcsmartspace/src/com/google/android/systemui/smartspace/BcSmartspaceView.java b/packages/SystemUI/bcsmartspace/src/com/google/android/systemui/smartspace/BcSmartspaceView.java
index 7b3d5e3c8ce4..3ac117583423 100644
--- a/packages/SystemUI/bcsmartspace/src/com/google/android/systemui/smartspace/BcSmartspaceView.java
+++ b/packages/SystemUI/bcsmartspace/src/com/google/android/systemui/smartspace/BcSmartspaceView.java
@@ -47,6 +47,7 @@ public class BcSmartspaceView extends FrameLayout implements BcSmartspaceDataPlu
     public final CardPagerAdapter mAdapter;
     public boolean mAnimateSmartspaceUpdate;
     public final ContentObserver mAodObserver;
+    public Handler mBgHandler;
     public int mCardPosition;
     public BcSmartspaceDataPlugin mDataProvider;
     public boolean mIsAodEnabled;
@@ -184,6 +185,10 @@ public class BcSmartspaceView extends FrameLayout implements BcSmartspaceDataPlu
         this.mAdapter.setDataProvider(this.mDataProvider);
     }
 
+    public void setBgHandler(Handler handler) {
+        this.mBgHandler = handler;
+    }
+
     public void onSmartspaceTargetsUpdated(List<? extends Parcelable> list) {
         int i;
         BaseTemplateCard baseTemplateCard;
-- 
2.49.0

